packages/lix-agent/scripts/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Determine the script directory
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+PROJECT_DIR="$(dirname "$SCRIPT_DIR")"
+MONOREPO_DIR="$(dirname "$(dirname "$(dirname "$PROJECT_DIR")")")"
+
+# Print setup message
+echo "Setting up Lix Agent development environment..."
+echo "Monorepo directory: $MONOREPO_DIR"
+echo "Project directory: $PROJECT_DIR"
+
+# Check if pnpm is installed
+if ! command -v pnpm &> /dev/null; then
+    echo "Error: pnpm is not installed"
+    echo "Please install pnpm first: npm install -g pnpm"
+    exit 1
+fi
+
+# Install dependencies
+echo "Installing dependencies..."
+cd "$MONOREPO_DIR" && pnpm install
+
+# Build the Lix SDK (required by the agent)
+echo "Building Lix SDK..."
+cd "$MONOREPO_DIR/lix/packages/lix-sdk" && pnpm run build
+
```

Comment:
I would remove the setup script. `pnpm --filter @lix-js/agent... build` builds all dependencies as they are used. Hardcoding this will lead to bugs.